.github/workflows/ci.yml

```diff
@@ -52,4 +52,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           CCM_CLUSTER_START_TIMEOUT_OVERRIDE: 600
-        run: mvn -B verify --file pom.xml -P it-${{ matrix.backend }}
+        run: mvn -B verify --file pom.xml -P it-${{ matrix.backend }},dse
```

Comment:
It would be nice to test without token and without the `dse` profile.

Reply:
What do you mean "without token"?

Reply:
Sorry, meant without a custom artifactory.

Reply:
Ah, sure. Was able to do that locally but didn't want to increase our build times by splitting this out more. In a follow up I can add some logic based on the `matrix.backend`.

Reply:
https://github.com/stargate/stargate/pull/218